src/RevokePassportTokens.php

```diff
@@ -35,6 +37,9 @@ public function handle(TokenRepository $tokenRepository)
         if ($user = $this->option('user')) {
             $query->where('user_id', $user);
             $this->info("Revoking all tokens for user $user...");
+        } elseif ($client = $this->option('client')) {
```

Comment:
This should probably handle the scenario where both user and client are passed in. I'd probably put this in its own if statement so we can add both queries if you pass both user and client